File: fizz.cs
public class Fizz {
  public static void Main() {
    for (var i = 0; i < 100; i++) {
      if (i % 15 == 0)
        System.Console.WriteLine("FizzBuzz");
      else if (i % 5 == 0)
        System.Console.WriteLine("Buzz");
      else if (i % 3 == 0)
        System.Console.WriteLine("Fizz");
      else
        System.Console.WriteLine(i);
    }
  }
}

File: fizz.m
#import <Foundation/Foundation.h>

int main()
{
  size_t i;
  for (i = 0; i < 100; i++) {
    if (i % 15 == 0) {
      NSLog(@"FizzBuzz");
    } else if (i % 5 == 0) {
      NSLog(@"Buzz");
    } else if (i % 3 == 0) {
      NSLog(@"Fizz");
    } else {
      NSLog(@"%d", i);
    }
  }
  return 0;
}
